<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inscription</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_title">
     <property name="styleSheet">
      <string notr="true">font: bold 14pt &quot;Segoe UI&quot;; color: #2c3e50;</string>
     </property>
     <property name="text">
      <string>Créer un compte</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="placeholderText">
      <string>Nom d'utilisateur</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Mot de passe</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_question">
     <property name="placeholderText">
      <string>Question secrète</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_reponse">
     <property name="placeholderText">
      <string>Réponse secrète</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_inscrire">
     <property name="styleSheet">
      <string notr="true">background-color: #2ecc71; color: white; padding: 6px; border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>S'inscrire</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
